react/week4/github-user-search/src/components/header.js

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Minor comment but try be consistent with your naming of your files with CamelCase or kebab-case (first is recommended I believe)